### ISSUE TITLE
[Accessibility] WCAG 2.1 2.1.1, 2.4.3, and 2.4.7 && Keyboard Usability

### ISSUE THREAD

**Issue by KimPlausible** [swagger-api/swagger-ui#5290]
### Q&A (please complete the following information)
 - OS: Windows
 - Browser: Chrome
 - Version: 73.0.3683.86
 - Method of installation: npm
 - Swagger-UI version: 3.17.6
 - Swagger/OpenAPI version: latest

### Describe the bug you're encountering

Aggregated accessibility bugs related to WCAG 2.1 2.1.1, 2.4.3, and 2.4.7 && Keyboard Usability
1. Keyboard focus is not visible/indicated for Cancel, Execute, and Clear buttons.
2. Unable to navigate within "Models" section by keyboard -- Tab navigation skips directly from deleteUser to validator.
3. Unable to navigate to Example Value/Edit Value or Models links via keyboard -- Tab navigation skips directly from "Try it out" to Parameter content type.
4. Keyboard focus remains outside of pop-up window (Available authorizations dialog box).
5. Keyboard focus is not clearly visible on Schemes combo box & Authorize button.
6. Keyboard focus on Expand/Collapse carrot is not visible.
7. Keyboard focus is not visible on "Try it out" button.
8. Keyboard focus is not clearly visible on "Response content type" combo box.
9. Keyboard focus not visible on "Close" button.

### To reproduce...

1. Navigate to https://petstore.swagger.io/#/pet/addPet -> Tab key over to "Try it out" button -> Enter key -> Tab key down to Execute -> Enter key -> Tab key over to Clear -> Enter key
2. Navigate to https://petstore.swagger.io/#/ -> Tab key through the entire page
3. Navigate to https://petstore.swagger.io/#/user/createUser -> Tab key to "Try it out" button -> Tab key
4. Navigate to https://petstore.swagger.io/#/pet -> Tab key to lock icon -> Enter key -> Tab key
5. Navigate to https://petstore.swagger.io/#/ -> Tab key to Schemes combo box -> Tab key to Authorize button
6. Navigate to https://petstore.swagger.io/ -> Tab key to pet -> Tab key to http://swagger.io link -> Tab key -> Enter key
7. Navigate to https://petstore.swagger.io/#/pet/addPet -> Tab key over to "Try it out" button -> Enter key
8. Navigate to https://petstore.swagger.io/#/pet/addPet -> Tab key down to "Parameter content type" combo box -> Tab key
9. Navigate to https://petstore.swagger.io/#/ -> Tab key to "Authorize" button -> Enter key -> Tab key down to "Close" button -> Enter key

### Expected behavior

1. Keyboard focus should be clearly visible/indicated for Cancel, Execute, and Clear buttons
2. Keyboard-only users should be able to access the content within "Models" section
3. Keyboard-only users should be able to access the clickable links Example Value/Edit Value and Models
4. Keyboard focus should be logical inside the dialog box -- should only iterate inside the dialog box and should not move out of it until & unless exited.​ Screen reader users will not be able to understand the flow of the focus inside and out of the dialog box.​
5. Keyboard focus should be clearly visible on Schemes combo box & Authorize button.
6. Keyboard focus on Expand/Collapse carrot should be visible.
7. Keyboard focus on "Try it out" button should be visible.
8. Keyboard focus on "Response content type" combo box should be clearly visible.
9. Keyboard focus on "Close" button should be visible.

### Screenshots

1. 
![image](https://user-images.githubusercontent.com/12721313/55605437-67e75400-5729-11e9-848e-1750dbc66976.png)
2. 
![image](https://user-images.githubusercontent.com/12721313/55605715-9ca7db00-572a-11e9-8177-1944cbd00b36.png)
3. 
![image](https://user-images.githubusercontent.com/12721313/55606169-a5011580-572c-11e9-8088-2a29a7e7cc80.png)
4. 
![image](https://user-images.githubusercontent.com/12721313/55608087-987fbb80-5732-11e9-88a6-3d0279ad8c7b.png)
5. 
![image](https://user-images.githubusercontent.com/12721313/55609232-0f1db880-5735-11e9-9797-143dd117d797.png)
6. 
![image](https://user-images.githubusercontent.com/12721313/55609438-89e6d380-5735-11e9-8a0c-81e78a43145c.png)
7. 
![image](https://user-images.githubusercontent.com/12721313/55609843-86a01780-5736-11e9-8346-eb04cca0b526.png)
8. 
![image](https://user-images.githubusercontent.com/12721313/55610100-12b23f00-5737-11e9-9be7-c6c3f89026df.png)
9. 
![image](https://user-images.githubusercontent.com/12721313/55610490-e3e89880-5737-11e9-868f-ecef6132d555.png)

**Comment by KimPlausible** [comment]
**Pri 0** (Blocking, high impact): 2, 3, 4
**Pri 2** (Not blocking, high impact): 1, 5, 7, 8, 9
**Pri 3** (Not blocking, low impact): 6

**Comment by msftedad** [comment]
Hi @KimPlausible, is there any update on this bug, and could you please provide a tentative ETA for fixing it?

**Comment by msftedad** [comment]
Hi @ponelat, is there any update on this bug and could you please provide a tentative ETA for fixing it.

**Comment by msftedad** [comment]
Hi @maggiewachs, this issue is still repro'ing in the latest version of Swagger UI(5.20.1). Could you please confirm if the timeline for fixing it has been finalized? Thanks

CC: @ponelat
